File: Net/Calculo_comision/Calculo_comision.cs
using System;

class Calculo_comision
{
    static void Main()
    {
        // Pregunta por el nombre del vendedor
        Console.Write("Ingrese su nombre por favor: ");
        string nombre = Console.ReadLine();

        // Pregunta por las ventas totales del mes
        Console.Write("Ingresa el total de ventas del mes: ");
        string ventasStr = Console.ReadLine();

        // Convierte el string de ventas a float
        float ventasTotales = float.Parse(ventasStr);

        // Calcula la comisión (20% de las ventas totales)
        float comision = ventasTotales * 0.20f;

        // Muestra el resultado
        Console.WriteLine($"Hola {nombre}, sus comisiónes este mes son de: {comision:C}");

        // Espera antes de cerrar
        Console.WriteLine("Presiona cualquier tecla para salir...");
        Console.ReadKey();
    }
}

File: Python/Calculo_comision.py
# Solicita el nombre del vendedor
nombre = input("Ingrese su nombre por favor: ")

# Solicita el total de ventas del mes
ventas_str = input("Ingresa el total de ventas del mes: ")

# Convierte la cadena de ventas a un número flotante
ventas_totales = float(ventas_str)

# Calcula la comisión (20% de las ventas totales)
comision = ventas_totales * 0.20

# Muestra el resultado
print(f"Hola {nombre}, sus comisiónes este mes son de: ${comision:.2f}")
